src/test/resources/30/single/opf/invalid/prefixes-002.opf

```diff
@@ -8,7 +8,6 @@
         <dc:language>la</dc:language>
         <dc:date>2011-09-01</dc:date>
         <meta property="dcterms:modified">2011-09-01T17:18:00Z</meta>
-		<meta property="meta-auth">Auth</meta>
```

Comment:
likewise, this test case may be used to check that properties in the default Vocabulary were recognized. To be verified, and if yes `meta-auth` should be replaced.